README.md

```diff
@@ -33,3 +33,18 @@ In order to provide more virtual and emulated resources with limited hardware re
 - Kubernetes cluster in virtual machines
 
 For ore detail design and information, please refer to the docs folder in this repository. 
```

Comment:
Can you please change the line from ore to more?

Reply:
sounds great! i thought it was an acronym and was trying to find this module for quite a while XD

Reply:
👍 thanks!